### PR TITLE
Add mini changelog to ISO week functions (see #957)

### PR DESCRIPTION
**Pull request by leshakoss**

### CODE REVIEW

File: src/addISOWeekYears/index.js [hunk 1]
```diff
@@ -16,6 +16,14 @@ import setISOWeekYear from '../setISOWeekYear/index.js'
  * ### v2.0.0 breaking changes:
  * 
  * - [Changes that are common for the whole library](https://github.com/date-fns/date-fns/blob/master/docs/upgradeGuide.md#Common-Changes).
+ * 
+ * - Function renamed:
```
Comment (kossnocorp) [inline src/addISOWeekYears/index.js]: The function was renamed from `addISOYears` to `addISOWeekYears`.

File: src/differenceInCalendarISOWeekYears/index.js [hunk 2]
```diff
@@ -15,6 +15,14 @@ import getISOWeekYear from '../getISOWeekYear/index.js'
  * 
  * - [Changes that are common for the whole library](https://github.com/date-fns/date-fns/blob/master/docs/upgradeGuide.md#Common-Changes).
  *
+ * - Function renamed:
```
Comment (kossnocorp) [inline src/differenceInCalendarISOWeekYears/index.js]: The function was renamed from `differenceInCalendarISOYears ` to `differenceInCalendarISOWeekYears `.

File: src/differenceInISOWeekYears/index.js [hunk 3]
```diff
@@ -18,6 +18,14 @@ import subISOWeekYears from '../subISOWeekYears/index.js'
  * 
  * - [Changes that are common for the whole library](https://github.com/date-fns/date-fns/blob/master/docs/upgradeGuide.md#Common-Changes).
  *
+ * - Function renamed:
```
Comment (kossnocorp) [inline src/differenceInISOWeekYears/index.js]: The function was renamed from `differenceInISOYears` to `differenceInISOWeekYears`.

File: src/endOfISOWeekYear/index.js [hunk 4]
```diff
@@ -18,6 +18,14 @@ import startOfISOWeek from '../startOfISOWeek/index.js'
  * 
  * - [Changes that are common for the whole library](https://github.com/date-fns/date-fns/blob/master/docs/upgradeGuide.md#Common-Changes).
  *
+ * - Function renamed:
```
Comment (kossnocorp) [inline src/endOfISOWeekYear/index.js]: The function was renamed from `endOfISOYear` to `endOfISOWeekYear`.

File: src/getISOWeekYear/index.js [hunk 5]
```diff
@@ -17,6 +17,14 @@ import startOfISOWeek from '../startOfISOWeek/index.js'
  * 
  * - [Changes that are common for the whole library](https://github.com/date-fns/date-fns/blob/master/docs/upgradeGuide.md#Common-Changes).
  *
+ * - Function renamed:
```
Comment (kossnocorp) [inline src/getISOWeekYear/index.js]: The function was renamed from `getISOYear` to `getISOWeekYear`.

File: src/subISOWeekYears/index.js [hunk 6]
```diff
@@ -16,6 +16,14 @@ import addISOWeekYears from '../addISOWeekYears/index.js'
  * 
  * - [Changes that are common for the whole library](https://github.com/date-fns/date-fns/blob/master/docs/upgradeGuide.md#Common-Changes).
  *
+ * - Function renamed:
```
Comment (kossnocorp) [inline src/subISOWeekYears/index.js]: The function was renamed from `subISOYears` to `subISOWeekYears`.

File: src/setISOWeekYear/index.js [hunk 7]
```diff
@@ -19,6 +19,14 @@ import differenceInCalendarDays from '../differenceInCalendarDays/index.js'
  * 
  * - [Changes that are common for the whole library](https://github.com/date-fns/date-fns/blob/master/docs/upgradeGuide.md#Common-Changes).
  *
+ * - Function renamed:
```
Comment (kossnocorp) [inline src/setISOWeekYear/index.js]: The function was renamed from `setISOYear` to `setISOWeekYear`.

File: src/lastDayOfISOWeekYear/index.js [hunk 8]
```diff
@@ -18,6 +18,14 @@ import startOfISOWeek from '../startOfISOWeek/index.js'
  * 
  * - [Changes that are common for the whole library](https://github.com/date-fns/date-fns/blob/master/docs/upgradeGuide.md#Common-Changes).
  *
+ * - Function renamed:
```
Comment (kossnocorp) [inline src/lastDayOfISOWeekYear/index.js]: The function was renamed from `lastDayOfISOYear` to `lastDayOfISOWeekYear`.

File: src/isSameISOWeekYear/index.js [hunk 9]
```diff
@@ -15,6 +15,14 @@ import startOfISOWeekYear from '../startOfISOWeekYear/index.js'
  * 
  * - [Changes that are common for the whole library](https://github.com/date-fns/date-fns/blob/master/docs/upgradeGuide.md#Common-Changes).
  *
+ * - Function renamed:
```
Comment (kossnocorp) [inline src/isSameISOWeekYear/index.js]: The function was renamed from `isSameISOYear` to `isSameISOWeekYear`.

File: src/addISOWeekYears/index.js [hunk 10]
```diff
@@ -14,9 +14,14 @@ import setISOWeekYear from '../setISOWeekYear/index.js'
  *
  *
  * ### v2.0.0 breaking changes:
- * 
+ *
  * - [Changes that are common for the whole library](https://github.com/date-fns/date-fns/blob/master/docs/upgradeGuide.md#Common-Changes).
  *
+ * - The function was renamed from `addISOYears` to `addISOWeekYears`.
+ *   "ISO week year" is short for [ISO week-numbering year](https://en.wikipedia.org/wiki/ISO_week_date).
```
Comment (kossnocorp) [inline src/addISOWeekYears/index.js]: Please also fix `"ISO week year" is >>a shortening<< for...` here and below

File: src/addISOWeekYears/index.js [hunk 11]
```diff
@@ -14,9 +14,14 @@ import setISOWeekYear from '../setISOWeekYear/index.js'
...
+ *   This change makes the name consistent with
+ *   locale-dependent week-numbering year helpers, e.g. `addWeekYears`.
```
Comment (kossnocorp) [inline src/addISOWeekYears/index.js]: Also, please add `,` after e.g.: `e.g.,`

File: src/addISOWeekYears/index.js (hunk 10 above)
Comment (leshakoss) [reply src/addISOWeekYears/index.js]: That's a correct usage.

> (followed by for) Of a word or phrase, constituting an abbreviation (for another) or shortened form (of another).
> “Phone” is short for “telephone” and "asap" short for "as soon as possible".

https://en.wiktionary.org/wiki/short#Adjective

File: src/addISOWeekYears/index.js (hunk 11 above)
Comment (leshakoss) [reply src/addISOWeekYears/index.js]: Fixed

File: src/addISOWeekYears/index.js (hunk 10 above)
Comment (kossnocorp) [reply src/addISOWeekYears/index.js]: Ok 👌

**Pull request merged**